### ISSUE TITLE
Command 'InstallPlugin' raised error

### ISSUE THREAD

**Issue by halega** [fman-community/fman#438]
**Version**
- fman 1.1.1
- Windows 10 1803

**Steps to reproduce**
1. Select "Install plugins" menu item.

**Expected**
List of available plugins. Or a friendly error message about network issues.

**Actual**
Error:
```
Command 'InstallPlugin' raised error.

Traceback (most recent call last):
  File "urllib\request.py", line 1254, in do_open
  File "http\client.py", line 1107, in request
  File "http\client.py", line 1152, in _send_request
  File "http\client.py", line 1103, in endheaders
  File "http\client.py", line 934, in _send_output
  File "http\client.py", line 877, in send
  File "http\client.py", line 1253, in connect
  File "http\client.py", line 849, in connect
  File "socket.py", line 712, in create_connection
  File "socket.py", line 703, in create_connection
ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "fman\impl\plugins\command_registry.py", line 52, in _execute_command
  File "C:\Users\kalashnikov_s\AppData\Local\fman\Versions\1.1.1\Plugins\Core\core\commands\__init__.py", line 1458, in __call__
    self._plugin_repos = find_repos(topics=['fman', 'plugin'])
  File "C:\Users\kalashnikov_s\AppData\Local\fman\Versions\1.1.1\Plugins\Core\core\github.py", line 11, in find_repos
    return list(map(GitHubRepo, _fetch_all_pages(url)))
  File "C:\Users\kalashnikov_s\AppData\Local\fman\Versions\1.1.1\Plugins\Core\core\github.py", line 15, in _fetch_all_pages
    data = _get_json(json_url + '&per_page=%d&page=%d' % (page_size, page))
  File "C:\Users\kalashnikov_s\AppData\Local\fman\Versions\1.1.1\Plugins\Core\core\github.py", line 70, in _get_json
    return json.loads(_get(url).decode('utf-8'))
  File "C:\Users\kalashnikov_s\AppData\Local\fman\Versions\1.1.1\Plugins\Core\core\github.py", line 73, in _get
    return urlopen(url).read()
  File "urllib\request.py", line 163, in urlopen
  File "urllib\request.py", line 466, in open
  File "urllib\request.py", line 484, in _open
  File "urllib\request.py", line 444, in _call_chain
  File "urllib\request.py", line 1297, in https_open
  File "urllib\request.py", line 1256, in do_open
urllib.error.URLError: <urlopen error [WinError 10061] No connection could be made because the target machine actively refused it>
```

**Comment by mherrmann** [comment]
Thank you for reporting this. I changed it now so you get a nicer message. Will go live with the next release in the coming days.

**Comment by mherrmann** [comment]
Implemented in [fman 1.1.2](https://fman.io/changelog#v1.1.2).
